utils/flags/flags.go

```diff
@@ -219,6 +219,12 @@ func GetCliFlags() []cli.Flag {
 			Usage:   "The S3/minio bucket to use when using S3 proxy backend.",
 			EnvVars: []string{"BAZEL_REMOTE_S3_BUCKET"},
 		},
+		&cli.IntFlag{
```

Comment:
Most of bazel-remotes other flags take string values, would you mind changing this to do that? We could use the values "dns", "path" and "auto". I think that's a bit easier to understand when reading a config file or command line arguments.

Reply:
Yes, your suggestion fine. I updated it.